lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -0,0 +1,367 @@
+package Genome::Model::Tools::Transcriptome::ErccMapUnaligned;
+
+use strict;
+use warnings;
+
+use Genome;
+use Path::Class;
+
+class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
+    is => 'Command::V2',
+    has_input => [
+        bam_file => {
+            is => 'Text',
+            doc => 'An aligned BAM file to a reference genome without the ERCC transcripts spiked in.',
+        },
+        ercc_fasta_file => {
+            is => 'Text',
+            doc => 'The FASTA format sequence for all ERCC transcripts.',
+            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa'],
```

Comment:
This is left from me.  It's unclear where to put this. I guess we could import this tiny FASTA as a reference sequence and a user could resolve a path for the FASTA in the build (rather than my temporary location).

The intent was for the user to have something they could copy/paste from the example until a more appropriate location for the FASTA was made.